### ISSUE TITLE
32 frames are selected continuously or randomly ？

### ISSUE THREAD

**Issue by miaoct** [selimsef/dfdc_deepfake_challenge#3]
Could you talk about it in detail ?
Thank you !

**Comment by miaoct** [comment]
And you say "Though only 400 videos can be used for validation as well." 
What is mean ? How to get tags for 400 videos and how to use them for verification ?
Thank you !

**Comment by selimsef** [comment]
You can have a look at the code for frame extraction here https://github.com/selimsef/dfdc_deepfake_challenge/blob/e6038f17fb16c9ee93f216a9292392158826d53d/kernel_utils.py#L53

**Comment by selimsef** [comment]
Regarding 400 videos I hardcoded their names. 
But originally (for the 4 models out of 5) I used all videos from the 0-2 folders, it just takes more time.  Left it like that to be reproducible.
Can be changed to use public validation set - not sure how it would affect final scores though. https://github.com/selimsef/dfdc_deepfake_challenge/blob/e6038f17fb16c9ee93f216a9292392158826d53d/training/datasets/classifier_dataset.py#L362
